index.cjs

```diff
@@ -4,7 +4,12 @@
 var path = require("path");
 
 // If in 'production' mode, use the combined/minified/optimized version of Cesium
-if (process.env.NODE_ENV === "production") {
+if (
+  process.env.NODE_ENV === "production" ||
+  process.env.NODE_ENV === "deploy-staging" ||
+  process.env.NODE_ENV === "deploy-production" ||
+  process.env.NODE_ENV === "deploy-dev"
```

Comment:
I think just `dev`, `staging` and `production`: https://github.com/PropellerAero/lambda-surface-math/blob/dev/codeship-services.yml#L19

Reply:
ok